src/views/SignUpLandingView.vue

```diff
@@ -83,7 +84,7 @@ export default {
   .btn-row {
     display: flex;
     justify-content: flex-end;
-    align-items: flex-end;
+    margin: 10px 0 10px 0;
```

Comment:
What do the CSS changes do?

Reply:
Nevermind, I can't read

Reply:
align-items just felt unnecessary and margin was because I though it looked bad not having any space between the end of the text and the button